exercises/sum-of-multiples/sum-of-multiples.spec.js

```diff
@@ -2,23 +2,23 @@ var SumOfMultiples = require('./sum-of-multiples');
 
 describe('SumOfMultiples', function () {
   it('to 1', function () {
-    expect(SumOfMultiples().to(1)).toBe(0);
+    expect(SumOfMultiples([3,5]).to(1)).toBe(0);
```

Comment:
Yes, I agree-- good catch.  I'll try to get to this tonight or first thing tomorrow.  Thanks!